src/components/v5/common/ActionSidebar/partials/forms/TransferFundsForm/utils.tsx

```diff
@@ -42,5 +50,9 @@ export const getTransferFundsPayload = (
       ? sanitizeHTML(annotationMessage)
       : undefined,
     customActionTitle: title,
+    isMultiSig: decisionMethod === DecisionMethod.MultiSig,
+    colonyRoles: extractColonyRoles(colony.roles),
+    colonyDomains: extractColonyDomains(colony.domains),
+    createdInDomain,
```

Comment:
Passing the createdInDomain so that if the action is a reputation motion, the saga can handle that properly. This was broken before.